guidelines/templates/resources/troubleshooting.md

```diff
@@ -1,19 +1,44 @@
 ---
 title: {Document title}
-type: Troubleshooting
 ---
 
->**NOTE:** Blockquotes in this document provide instructions. Remove them from the final document.
->
->This is a template for the **Troubleshooting** document type that explains all details needed for Kyma and its components' troubleshooting.
->
->In the document:
->
->- Follow the `10-{00}-{document-title}.md` convention to name the document.
->- The title must summarize what the document is about. You can name it after a particular problem it solves. For example, if a user gets an error called "Transport is closing" when they run a Helm command, you can entitle the troubleshooting document **"Transport is closing" error**. To learn more about how to use quotation marks in metadata, see the document devoted to the [Markdown format](https://kyma-project.io/docs/components/service-catalog/#console-ui-views-specifications-in-the-console-ui-markdown-documents).
->- In the body of the document:
->   - Describe the problem from the user's perspective.
->   - Explain the problem's possible cause, if needed.
->   - Provide a solution.
->
-> For reference, see the existing **Troubleshooting** [documents](https://kyma-project.io/docs/root/kyma/#troubleshooting-troubleshooting) for Kyma.
+<!-- Use this template to write Troubleshooting instructions. You can also use it to document alarm clearing strategies.
+
+As document file name, follow the pattern {COMPONENT_ABBRV}-{NUMBER_PER_COMPONENT}-{FILE_NAME}.md}.
+
+As document title, mention the symptom that needs fixing ("Cannot access...") or the error message. To quote an error message, start and end with `'` to escape `"`. for example, `title: '"FAILED" status for created ServiceInstances'`. Do not use the cause as title ("Incompatible version"), because we also want to help users who have no idea about the cause and only know something's not working as expected.
+
+It's good practice to use three standard headlines (like “Condition”, “Cause”, “Remedy”), each might have just one sentence or more as needed. 
+-->
+
+## Symptom
+
+<!-- Describe the problem from the user's perspective. Provide an undesirable condition or symptom that the user may want to correct. This could be an error message or an undesirable state.
```

Comment:
```suggestion
<!-- Describe the problem from the user's perspective. Provide the undesirable condition or symptom that the user may want to correct. This could be an error message or an undesirable state.
```